### PR TITLE
Add Akiflow 2.28.11

### PR DESCRIPTION
**Pull request by rpowis**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### Notes

I get the following from `brew audit` but I can't find anywhere to pull the version from for a `livecheck` stanza.

```
- Version '2.28.11' differs from '' retrieved by livecheck.
```

### CODE REVIEW

**Comment by rpowis** [comment]
ah yes. exactly what i was talking about under the notes section of my PR description. i'm not sure how best to resolve that so any pointers welcome. 
https://github.com/Homebrew/homebrew-cask/actions/runs/3143696595/jobs/5108770398#step:11:14

File: Casks/akiflow.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask "akiflow" do
+  version "2.28.11"
+  sha256 "2d22398a44fe581f7177707df319d3ba35e1b805b4a9206b958f6583c86f8717"
+
+  url "https://akiflow.com/releases/Akiflow-#{version}-universal.dmg"
+  name "akiflow"
+  desc "Time Blocking platform to save 2 hours every day"
+  homepage "https://akiflow.com/"
+
```
Comment (p-linnane) [inline Casks/akiflow.rb:9]:
```suggestion

  livecheck do
    url "https://akiflow.com/releases/download"
    strategy :header_match
  end

```

Comment (rpowis) [reply Casks/akiflow.rb:9]: ah ha! nice find. thanks very much @p-linnane 🎉

**Comment by p-linnane** [comment]
Can you please try to add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap) for this cask? Thanks.

**Comment by rpowis** [comment]
done @p-linnane. let me know if there's anything else 👍

**Comment by p-linnane** [comment]
Looks good. Thank you!

**Comment by miccal** [comment]
Thank you @rpowis.

**Review by miccal — APPROVED** [review]

**Pull request merged**